main.css

```diff
@@ -163,6 +196,7 @@ h3.headline {
 @media screen and (min-width: 60em) {
   .program__row {
     margin-bottom: 3rem;
+    width: 975px;
```

Comment:
This seems funky to me. Let's discuss what you're intending to accomplish with this.